src/ResultsActivity/ResultsActivity.js

```diff
@@ -196,6 +196,12 @@ class ResultsActivity extends Component {
       return showControls
     }
 
+    const showRemainingButton = activities => {
+      const attendanceInfo = activities.filter(activity => activity.attendanceInfo)
+      if (totalPaid !== 0 || attendanceInfo.length) return true
+      return false
```

Comment:
No need for the `if`.  Return `totalPaid !== 0 || attendanceInfo.length`

Reply:
Aye, sorted dude.